types/mdx/types.d.ts

```diff
@@ -46,7 +46,7 @@ export interface MDXProps {
 /**
  * The type of the default export of an MDX module.
  */
-export type MDXContent = FunctionComponent<MDXProps>;
+export type MDXContent = (props: MDXProps) => JSX.Element;
```

Comment:
Given that `FunctionComponent` is not exported, this change is good. Otherwise it might break other folks.

Should `Clascomponent` also be nullifiable?

Reply:
I think the `render` method return value should, but that's up to the JSX framework to implement.